ci-operator/step-registry/ocm/e2e/kind/create/ocm-e2e-kind-create-commands.sh

```
@@ -58,6 +58,10 @@ echo "Waiting up to 5 minutes for VM to be ready"
 _timeout=300
 _elapsed=''
 _step=15
+
+# make sure the key has correct permissions
+chmod 600 $KEY
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)